app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckContactDetailsMrnController.scala

```diff
@@ -110,13 +110,13 @@ class CheckContactDetailsMrnController @Inject() (
               BadRequest(renderTemplate(updatedForm, fillingOutClaim, router, mandatoryDataAvailable))
             },
             {
-              case answer @ Yes =>
+              case Yes =>
                 Redirect(
                   router.CheckAnswers.when(fillingOutClaim.draftClaim.isComplete)(alternatively =
-                    router.nextPageForChangeClaimantDetails(answer, featureSwitch)
+                    router.nextPageForChangeClaimantDetails(Yes, featureSwitch)
```

Comment:
Also I don't like that Router hiding implementation details from me, it is almost like to understand complete picture I need to go and check multiple places.
I always liked to see hole picture in one single place.